src/components/member/AvatarSetting.js

```diff
@@ -38,7 +38,7 @@ const AvatarSetting = ({ user }) => {
   const classes = useStyles();
   const { mutate: onUploadAvatar } = useMutation(MUTATION_KEYS.UPLOAD_AVATAR);
 
-  const userId = user.get('id');
+  const userId = user.id;
```

Comment:
I see there's a conflict between undefined or not for the user, (not you fault tho) maybe let's add a `?.` here too.